package-lock.json

```diff
@@ -1,4062 +1,8 @@
 {
```

Comment:
Perhaps this is a matter of npm version you're using, but it'd be the best if we both use a version that generates a package-lock.json with lockfileVersion 2.